Firmware/variants/MK25-RAMBo10a.h

```diff
@@ -184,6 +184,9 @@
 #endif
 #define BED_MAXTEMP 125
 
+//Lowertemp
+#define HEATER_LOWER_TEMP 50 //lower hotend temperature by 50°C
```

Comment:
I think this new setting should be off by default so that we do not change the default behavior. Users who know what they are doing can then change this setting on their own in my opinion.

----
I feel this description could be more detailed. This doesn't really tell me what the setting does, and I would have look at the source code to understand. They way I see it, the comment/description here is basically our documentation.

Suggested description:
```
Uncomment to change the maximum allowed temperature drop while a print is paused. The default behavior sets the target temperature to 0°C.
Example: HEATER_LOWER_TEMP = 10 and target temperature is 215°C when the print was paused. The target temperature will then be set to 205°C instead of 0°C.
```

Reply:
Also agree with off by default. Was a bit confused about the feature until I read the source code for the PR (recall discussion from last meeting). I like the description provided by @gudnimg 👍